### ISSUE TITLE
Create 3D triangulation from cloud of points

### ISSUE THREAD

**Issue by iojea** [PyMesh/PyMesh#196]
Hi,
I am hoping to use pymesh for building finite element tetrahedral meshes, beginning only with the cloud of points. 
I've done this using tetgen (the original library), in the following way: 
1.  I run:  './tetgen my_points.node', obtaining my_points.1.ele (and .1.edge, .1.face, .1.node). 
2. If necessary, I run './tetgen -q...'  on the result of 1) for improving the mesh quality. 
Step 1) is just a Delaunay tetradrahelization, but tetgen is much faster than scipy.sparse.Delaunay. In fact, this is the main reason why I begin using tetgen. 
I though pymesh would allow me to avoid the usage of tetgen as a separate entity, integrating the mesh generator to my Python code. However, I can't find how to perform step 1). Every example that I found builds a tetrahedral mesh from another one or, at least from a triangular one. 
Is the functionality of tetgen describen in 1) available in pymesh? Is there any other way of obtaining an initial mesh, with the cloud of points as sole input?
Thanks in advance,
Ignacio.

**Comment by Cascades** [comment]
I don't believe PyMesh currently supports tetgen for 3D meshes, here is the [tegen stuff for PyMesh](https://pymesh.readthedocs.io/en/latest/api_mesh_generation.html?highlight=tetgen#tetrahedralization).

There are a variety of other tetgen wrapper for python I can find online though such as [this](https://pypi.org/project/tetgen/). Might these be of any use?
